DataScience/ds.config

```diff
@@ -10,10 +10,3 @@ ExperimentalUnitDurationDays: 1
 [Experimentation]
 LearningRates: 0.5,0.25,0.1,0.05,0.025,0.01,0.005,0.0025,0.001,0.0005,0.00025,0.0001,0.00005,0.000025,0.00001
 RegularizationValues: 0,1e-1,1e-2,1e-3,1e-4,1e-5,1e-6,1e-7,1e-8,1e-9
-
-[Visualization]
-LogDownloaderURL: https://cps-staging-exp-experimentation.azurewebsites.net/api/Log?account={ACCOUNT}&key={KEY}&start={START}&end={END}&container={CONTAINER}
```

Comment:
why is this removed?

Reply:
It's moved to AzureStorageDownloader.py

After a discussion with Alekh, he suggested avoiding having it in ds.config to. Since this is only used on AzureStorageDownloader.py, I moved it there. I don't have a strong feeling about this, so we can go either way with this